functions/HttpTriggerAPIUsersIdTask/__init__.py

```diff
@@ -1,24 +1,67 @@
 import logging
-
+import pyodbc
+import os
 import azure.functions as func
+import datetime
+import json
+import time
+
 
 
+# to handle datetime with JSON
+def default(o):
+    if isinstance(o, (datetime.datetime, datetime.date)):
+        return o.isoformat()
+
 def main(req: func.HttpRequest) -> func.HttpResponse:
-    logging.info('Python HTTP trigger function processed a request.')
+    
+
+    # define the server and database names
+    server = os.environ["ENV_DATABASE_SERVER"]
+    database = os.environ["ENV_DATABASE_NAME"]
+    username = os.environ["ENV_DATABASE_USERNAME"]
+    password = os.environ["ENV_DATABASE_PASSWORD"]
+    
+    # define the connection string
+    driver = '{ODBC Driver 17 for SQL Server}'
+    cnxn = "Driver={};Server={};Database={};Uid={};Pwd={};Encrypt=yes;TrustServerCertificate=no;Connection Timeout=30;".format(
+        driver, server, database, username, password)
+    
+   
+    id = req.params.get('userId')
 
-    name = req.params.get('name')
-    if not name:
+    if not id:
+        return func.HTTPResponse('Bad Request', status_code=400 )
+    else:
         try:
-            req_body = req.get_json()
-        except ValueError:
-            pass
+            with pyodbc.connect(cnxn) as conn:
+                return tasks_query(conn, id)

```

Comment:
Is this going to be the only tasks query? Maybe something like get_tasks_by_id?

Reply:
Yeah, I would agree with Lenny, this would clarify what we actually getting